examples/all-clusters-minimal-app/nrfconnect/README.md

```diff
@@ -55,17 +58,23 @@ and [Zephyr RTOS](https://zephyrproject.org/). Visit Matter's
 [nRF Connect platform overview](../../../docs/guides/nrfconnect_platform_overview.md)
 to read more about the platform structure and dependencies.
 
-The Matter device that runs the all clusters application is controlled by the
-Matter controller device over the Thread protocol. By default, the Matter device
-has Thread disabled, and it should be paired with Matter controller and get
-configuration from it. Some actions required before establishing full
-communication are described below.
+The sample uses buttons for changing the lock and device states, and LEDs to show the state of these changes. You can test it in the following ways:
+
+- Standalone, using a single DK that runs the door lock application.
+
+- Remotely over the Thread or the Wi-Fi protocol, which in either case requires more devices, including a Matter controller that you can configure either on a PC or a mobile device.
+
+By default, the Matter accessory device has IPv6 networking disabled. You must pair it with the Matter controller over Bluetooth® LE to get the configuration from the controller to use the device within a Thread or Wi-Fi network. You have to make the device discoverable manually (for security reasons). The controller must get the commissioning information from the Matter accessory device and provision the device into the network.
```

Comment:
Also about this line: we are adding information about manual discovery, but there is no mention of this requirement in the Testing section or elsewhere in the document, and so even the UI instructions are confusing. I'd suggest adding the relevant information to the Testing section.

Reply:
You can find it in the `Bluetooth LE advertising` section. Maybe it's worth to add a link to that section in this sentence.